### PR TITLE
[BOYSCOUT] FIX reverse import by creating a Component Object Model

### PR DESCRIPTION
**Pull request by gabrielagabriel**
## Description

Avoid importing index on SuccessModal

## What has been done

Export **SuccessModal Elements** for testing purposes

## Things to consider

- Styled-components syntaxe

## How was it was tested

Local

### CODE REVIEW

File: src/successModal/elements.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+const SuccessAction = styled.div`
+  padding: ${space.xl};
+`
+
+const SuccessButton = styled(Button)`
+  justify-content: center;
+
+  & {
+    min-width: 8rem;
+  }
```
Comment (simonrelet) [inline src/successModal/elements.tsx:99]: I think the `& {}` part can be removed to have:
```css
justify-content: center;
min-width: 8rem;
```

File: src/successModal/index.tsx [hunk 2]
```diff
@@ -1,115 +1,6 @@
...
-const SuccessModalStyle = {
-  StyledSuccessModal,
-  Media,
-  Figure,
-  Content,
-  SuccessTitle,
-  SuccessAction,
-  SuccessButton,
-}
+import SuccessModalElements from './elements'
```
Comment (simonrelet) [inline src/successModal/index.tsx:2]: You can simplify the import/export to:

```ts
export { default, SuccessModalProps } from './SuccessModal'
export { default as SuccessModalElements } from './elements'
```

File: src/successModal/elements.tsx (hunk 1 above)
Comment (gabrielagabriel) [reply src/successModal/elements.tsx:99]: not for `min-width` & here increase specificity

Comment (gabrielagabriel) [reply src/successModal/elements.tsx:99]: @simonrelet  I couldn't remove it from Button and that's why I'm increasing specificity, better than a important, for sure 🙅‍♀️

**Pull request merged**